website/client/components/achievements/chooseClass.vue

```diff
@@ -19,7 +19,9 @@
                 :withBackground='false',
                 :overrideAvatarGear='classGear(heroClass)',
                 :hideClassBadge='true',
-                :spritesMargin='"1.8em 1.5em"',
+                :spritesWidth='spriteSize(heroClass).width'
```

Comment:
instead of using 3 different props, can't just just pass spriteSize once and then access it as spriteSize.width, ... in the child component? And actually why are you passing this through a prop since it's used only in one child component? instead of calculating it directly in the child component?